### ISSUE TITLE
Ways to prevent overfitting in YOLOv3?

### ISSUE THREAD

**Issue by MicahImani** [AlexeyAB/darknet#4795]
Hello @AlexeyAB,

I'm currently working on a fish detection dataset. I've looked into modifying the loss function with L1 and L2 Regularization to prevent overfitting. Would this be effective? Could you recommend any ways to modify the loss function or anything I could look into?
